bin/counts_summary.Rmd

```diff
@@ -340,7 +340,7 @@ df_PCA %>%
         title = "PCA, first three principal components",
         subtitle = "Point size encodes PC3", x = "PC1", y = "PC2"
     ) +
-    custom_theme(legend.position = 0, aspect = 1) +
```

Comment:
`aspect` should not be removed, can be replaced by `aspect.ratio = 1`

Reply:
I added `aspect.ratio = 1`